starter-code/pizza.js

```diff
@@ -1 +1,81 @@
+'use strict'
 // Write your Pizza Builder JavaScript in this file.
+
+
+var $greenPep = $('div#pizza section.green-pepper'); 
+
+var $pepperonni = $('div#pizza section.pep'); 
+
+var $mushrooms = $('div#pizza section.mushroom'); 
+
+var $crust = $('.crust');
+
+var $sauce = $('.sauce');
+
+
+// var $adReButton = $.each($('button'), function (){});
+
+var $buttonPe = $($('button')[0]);
+
+var $buttonMu = $($('button')[1]);
+
+var $buttonGr = $($('button')[2]);
+
+var $buttonCr = $($('button')[4]);
+
+var $buttonSa = $($('button')[3]);
+
+// function addRemove () {
+//     var numb = 0;
+//     if (numb = 0) {
+//         $pepperonni.remove()
+//         numb = 1;
+//     } else {
+//         $pepperonni.add()
+//     }
+// }
+
+function removePe () {
+    if ($pepperonni.is(':visible')) {
+        $pepperonni.hide();
+    } else if ($pepperonni.is(':hidden')){
+        $pepperonni.show();
+    }
+}
+
+function removeMu () {
+
+    if ($mushrooms.is(':visible')) {
+        $mushrooms.hide();
+    } else if ($mushrooms.is(':hidden')){
+        $mushrooms.show();
+    }
+}
+
+function removeGr () {
```

Comment:
Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.